Xiaomi/Mi11T/AndroidManifest.xml

```diff
@@ -0,0 +1,10 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="me.phh.treble.overlay.xiaomi.mi11t"
+        android:versionCode="1"
+        android:versionName="1.0">
+        <overlay android:targetPackage="android"
+                android:requiredSystemPropertyName="ro.product.vendor.device"
+                android:requiredSystemPropertyValue="amber"
```

Comment:
I'm sorry, but this name is way too generic, it's highly likely other OEMs have/will give that name
You'll need to find another way to match the device

Reply:
Alright, will do